app/main.py

```diff
@@ -1,7 +1,46 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+from requests import Response
+
+
+API_KEY = os.environ.get("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+
+
+def get_weather(city: str = "Paris") -> Response:
+    print(f"Requesting {city} weather from Weather API...")
+
+    params = {"key": API_KEY, "q": city, "aqi": "no"}
+    request = requests.get(URL, params=params).json()
+
+    if "error" in request:
+        print(request["error"]["message"])
+        return request
```

Comment:
I'd recommend to check status code instead of error attribute that may be changed in the future